cli/module_build.go

```diff
@@ -159,6 +161,29 @@ func (c *viamClient) moduleBuildListAction(cCtx *cli.Context) error {
 	return nil
 }
 
+// filterMap is a helper that returns a new map based on k,v pairs that pass predicate.
+func filterMap[K comparable, V any](orig map[K]V, predicate func(K, V) bool) map[K]V {
```

Comment:
Like the generic approach. Lets move to a utils.go file so we can reuse in the future?

Reply:
done in 40ad58963

Reply:
LGTM!